parser-typechecker/package.yaml

```diff
@@ -180,3 +158,27 @@ executables:
       - unison-util
       - unison-util-relation
       - unison-pretty-printer
+
```

Comment:
Just moved this to the bottom because it's the most boring clause, I think. Also, I may have added an extension or two, for consistency with other packages.